src/emulator/util/src/util.cpp

```diff
@@ -0,0 +1,46 @@
+#include <util/string_convert.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#include <processenv.h>
+#include <shellapi.h>   // CommandLineToArgvW
+#else
+#include <unistd.h>
+#endif
+
+#include <iostream>
+#include <memory>
+#include <string>
```

Comment:
I think these includes aren't needed, and `<string>` should be included in the string_covert header.

Reply:
`iostream` is needed for the `std::cerr`, won't be needed once the logging stuff's in.